### ISSUE TITLE
timeout string like "10s" not parsed any more on python 3.6.9

### ISSUE THREAD

**Issue by ldjebran** [RedHatQE/wait_for#24]
When using strings like "10s"  on python 3.6.9 an exception is raised

```python
>>> from wait_for import  _parse_time
>>> _parse_time('12s')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/cloud-user/py3/lib/python3.6/site-packages/wait_for/__init__.py", line 28, in _parse_time
    parsed, code = calendar.parse(t)
  File "/home/cloud-user/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 1844, in parse
    retS, retTime, matched = parseMeth(s, sourceTime)
  File "/home/cloud-user/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 1408, in _partialParseQUnits
    sourceTime = self._evalQUnits(parseStr, sourceTime)
  File "/home/cloud-user/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 1100, in _evalQUnits
    sourceTime = self._buildTime(sourceTime, quantity, modifier, units)
  File "/home/cloud-user/py3/lib/python3.6/site-packages/parsedatetime/__init__.py", line 389, in _buildTime
    ctx.updateAccuracy(realunit)
  File "/home/cloud-user/py3/lib/python3.6/site-packages/parsedatetime/context.py", line 137, in updateAccuracy
    acc = self._ACCURACY_REVERSE_MAPPING[acc]
KeyError: 's'
```

This is working as expected on python 3.6.8

**Comment by ldjebran** [comment]
Fix is addressed https://github.com/bear/parsedatetime/pull/239

**Comment by RonnyPfannschmidt** [comment]
fixed by #30

**Comment by RonnyPfannschmidt** [comment]
@quarckster please close this one as fixed
